### PR TITLE
Add list of tested devices

### PR DESCRIPTION
**Pull request by Camerooooon**
I will also add a new file full of the testing requirements (Like a checklist) before this PR gets merged.

### CODE REVIEW

**Review by JakeRoggenbuck — APPROVED** [review]
Yes!

**Review by Shuzhengz — APPROVED** [review]
I added mine too

File: README.md [hunk 1]
```diff
@@ -48,6 +50,22 @@ Note: The latest release of acs can also be installed locally with the following
...
+| ----------- | ---------- |
+| Dell XPS 13 9360 | Working |
+| Dell Latitude 7480 | Working |
+| Steam Deck | Working |
+| Thinkpad T400 | Working |
+| Thinkpad X230 | Working |
+| Thinkpad W540 | Working |
+| ThinkPad X1 Extreme Gen 1 | Working |
+| Thinkpad P1 Gen 4 (Intel Core) | Working |
+| Thinkpad P14 Gen 2 (AMD) | Working |
```
Comment (Shuzhengz) [inline README.md:66]: Might wanna set this to partially working though (idt we figured out the clockspeed issue yet)

Comment (JakeRoggenbuck) [reply README.md:66]: True

**Review by JakeRoggenbuck — APPROVED** [review]

**Comment by Camerooooon** [comment]
I got a ThinkPad X220 we can try, it's mega corrupt harddrive though so very unstable.

**Comment by Shuzhengz** [comment]
> @Shuzhengz  is https://github.com/JakeRoggenbuck/auto-clock-speed/issues/443 related to your Mostly Working rating? If so I'll add it to the notes column.

yep

**Comment by Camerooooon** [comment]
Oh cool I did not know that you could reference issues in commit messages.

**Comment by Camerooooon** [comment]
Re-review with the changes and then ready for merge? @JakeRoggenbuck @Shuzhengz

**Review by Shuzhengz — APPROVED** [review]

**Review by JakeRoggenbuck — APPROVED** [review]

**Pull request merged**